src/pages/Main/Main.js

```diff
@@ -1,11 +1,88 @@
-import React from 'react';
+import React, { Component } from 'react';
+
+import MainImgBar from './MainImgBar/MainImgBar';
+import FoodThemeListSection from './FoodThemeListSection/FoodThemeListSection';
+import PopStoreSection from './PopStoreSection/PopStoreSection';
+
+import './Main.scss';
+
+const slideWidth = 1500;
 
 class Main extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      slideNum: 1,
+      slideTranslate: 0,
+      popStore: [],
+      bestListImg: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://10.58.0.218:8000/restaurants/sub_categories')
```

Comment:
```suggestion
    fetch(`${API}/restaurants/sub_categories`)
```
위와 같은 형태로 src/config.js 위치에 end point 분리해서 관리해주세요! 나머지 유사한 케이스도 마찬가지로 적용해주세요

Reply:
넵!